front-end/src/components/User/SetFilterCategory.js

```diff
@@ -4,33 +4,34 @@ import CenterConsole from '../CenterConsole/CenterConsole.js';
 
 import axios from 'axios';
 
-//friendData are an array of the friend's recipes
-
 class SetFilterCategory extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       currCategory: undefined,
-      friendData: undefined,
       userId: '',
       userUrlRecipes: []
-
     };
 
     this.setCurrCategory = this.setCurrCategory.bind(this);
-
   }
 
   componentDidMount() {
-    console.log('this is component did mount');
-    console.warn('these are props: ', this.props);
-    this.setState({userId: this.props.userId});
+    //console.log('Props in componentDidMount: ', this.props);
```

Comment:
Is this just for future debugging?

Reply:
Just in case :P I'll remove it once connected with center console!